Modules/Core/Libraries/Sumex.php

```
@@ -222,12 +218,12 @@ public function __construct(array $params)
         ];
 
         $esrTypes       = ['9', 'red'];
-        $this->_esrType = $esrTypes[$CI->mdl_settings->setting('sumex_sliptype')];
+        $this->_esrType = $esrTypes[\Modules\Core\Models\Setting::getValue('sumex_sliptype') ?? 0];
```

PHPMD (2.15.0)

Avoid using static access to class '\Modules\Core\Models\Setting' in method '__construct'. (undefined)

(StaticAccess)